demisto_sdk/commands/common/docker_helper.py

```diff
@@ -346,9 +347,12 @@ def create_image(
             the new created image
         Flow:
             1. creating a container using an existing image
-            2. running the istallation scripts
+            2. running the installation scripts
             3. committing the docker changes (installed packages) to a new local image
         """
+        logging_setup()
```

Comment:
is this not called in the `pre-commit` command? 🤔

Reply:
it's called that, but I added this because I didn't see the logs.

anyway I will reverting this